goss.hcl

```diff
@@ -0,0 +1,48 @@
+description = "Goss - Quick and Easy server testing/validation"
+binaries = ["goss"]
+homepage = "https://goss.rocks/"
+source = "https://github.com/goss-org/goss/releases/download/v${version}/goss-${release}${os}-${arch_}"
+vars = {
+  "arch_": "${arch}"
+}
+
+platform "darwin" {
+  source = "https://github.com/goss-org/goss/releases/download/v${version}/goss-alpha-${os}-${arch_}"
+
+  on "unpack" {
+    rename {
+      from = "${root}/goss-alpha-${os}-${arch_}.exe"
+      to = "${root}/goss.exe"
+    }
+  }
+}
+
+platform "darwin" "arm64" {
+  vars = {
+    "arch_": "amd64",
+  }
+}
```

Comment:
The assets on the latest goss release (https://github.com/goss-org/goss/releases/tag/v0.3.21) does not have a `darwin` `arm64` release.

I looked around at a few other packages and found at least one example of using `amd64` when `arm64` was not present.

https://github.com/cashapp/hermit-packages/blob/cc23198eeed31c9b92de117d989b25b846541ee8/atlas.hcl#L11

My understanding is that `arm64` is M1/M2 macs and that there is a program to allow running amd64 binaries on that machine: https://en.wikipedia.org/wiki/Rosetta_(software).